### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by piercus**
Hello Benjamin,

Please check this, to summarize my remarks are : 
* the output file were not a readable json, i have fixed it
* added clean up stuff in .gitignore and Readme by doing the install on my side, I feel a lot more could be done, so please do not consider this as a full review, more as my personnal "by the way" cleaning
* Appart from being readable, i suggest we try to follow the JSON-related convention for the output, in JSON it makes more sense to put the augmentedMusic as an object and to not put the keys when NaN
See other remaks for discussion in the code

### CODE REVIEW

File: bin/augment_race_music.py [hunk 1]
```diff
@@ -241,32 +241,44 @@ def update_inputs(historic, input, output):
...
-                except:
-                    # Si le cheval est absent de la bdd des musiques augmentées
-                    augmented_music = [
-                        [float(clean_results_position(result)), np.NaN, np.NaN]
-                        for result in clean_music_to_list(horse["musique"])
-                    ]
-                horse["augmented_music"] = str(list(augmented_music))
+                augmented_music = []
+                
+                if id in historic:
```
Comment (piercus) [inline bin/augment_race_music.py:258]: I have just make reformatting for now, but I feel historic here should be filter to only contain past information, not all of them do you agree ?

File: bin/augment_race_music.py [hunk 2]
```diff
@@ -241,32 +241,44 @@ def update_inputs(historic, input, output):
...
+                  for histo in historic[id]:
+                    position = int(histo[0])
+                    item = {"position": position}
+                    if math.isnan(position):
+                      raise Exception('no position')
+                    
+                    if not math.isnan(histo[1]):
+                      item['priceFirst'] = histo[1]
+                      
+                    if not math.isnan(histo[2]):
```
Comment (piercus) [inline bin/augment_race_music.py:269]: I'm not clear why date is NaN, it should always be not NaN

File: bin/augment_race_music.py [hunk 3]
```diff
@@ -241,32 +241,44 @@ def update_inputs(historic, input, output):
...
-                horse["augmented_music"] = str(list(augmented_music))
+                augmented_music = []
+                
+                if id in historic:
+                  
+                  for histo in historic[id]:
+                    position = int(histo[0])
+                    item = {"position": position}
+                    if math.isnan(position):
+                      raise Exception('no position')
```
Comment (piercus) [inline bin/augment_race_music.py:264]: Here i have raise an error for personnal understanding, but i feel that when the horse is disqualified, there should be no exception, but i did not face any of this situation in the base.

Are you filtering the disqualified archive rows earlier in the code ? I feel we should not

File: bin/augment_race_music.py [hunk 4]
```diff
@@ -241,32 +241,44 @@ def update_inputs(historic, input, output):
...
                 dai = False
                 # update augmented music databases with this race's result
                 try:
                     result = horse["results"]["position"]
                 except:
                     dai = True
-                performance = [result, priceFirst, date]
+                    result = None
+                
+                # performance = {"position": result, "priceFirst": priceFirst, "date": date, "dai": dai}
```
[the remark this answers is not in the document]
Comment (benjamrio) [reply bin/augment_race_music.py:254]: You are right. My bad.

File: bin/augment_race_music.py (hunk 1 above)
Comment (benjamrio) [reply bin/augment_race_music.py:258]: agreed.

File: bin/augment_race_music.py (hunk 3 above)
Comment (benjamrio) [reply bin/augment_race_music.py:264]: Yes I was filtering them out, as we did in our earlier project. I agree that is would be more intersting to keep them

File: bin/augment_race_music.py (hunk 2 above)
Comment (benjamrio) [reply bin/augment_race_music.py:269]: I think I had some troubles when running the scripts on some files.

**Pull request merged**